tests/integration-tests/README.md

```diff
@@ -0,0 +1,532 @@
+# Integration Tests
+
+This directory contains the integration tests for the Open Enterprise Agent (OEA).
+
+## Main concepts
+
+The integration tests are written in Kotlin, and use the following tools and libraries:
+
+1. [Serenity BDD](https://serenity-bdd.github.io/) for test execution engine
+2. [Hoplite](https://github.com/sksamuel/hoplite) for configuration management
+3. [Ktor](https://ktor.io/) for HTTP listener (async receiver for webhook messages)
+4. [PRISM Kotlin client](https://github.com/hyperledger-labs/open-enterprise-agent/packages/1919198) for OEA API models.
+5. [Atala Automation](https://github.com/input-output-hk/atala-automation/) for general testing helpers.
+6. [Awaitility](http://www.awaitility.org/) for asynchronous operations waiting.
+7. [TestContainers](https://www.testcontainers.org/) for Docker containers management.
+
+The project uses Behavior Driven Development (BDD) approach, and the tests are written with the help of Gherkin language.
+The key benefits of the BDD approach are:
+
+* BDD encourages collaboration between developers, testers, and non-technical stakeholders (such as business analysts or product owners).
```

Comment:
* BDD encourages collaboration between developers, testers, and non-technical stakeholders (business analysts or product owners).